UKHomeOffice/sgmr-service
# COP-2143: Add password reset functionality

## Description

Add functionality to reset passwords.

## Testing

Please note, the API changes are still in the PR stage so please make sure you check out a correct branch before testing.

1. Go to the sign-in form
2. Click `Forgotten your password?`
3. Enter your email address
4. Retrieve the token either by checking the API logs or your inbox, you might have to replace the DEV base url with your localhost to keep only the URL path with query string, i.e. `/new-password?token=123`
5. Open the new password URL => `http://localhost:8080/new-password?token=TOKEN_RETRIEVED_IN_PREVIOUS_STEP`
6. Enter your new password and submit
7. You password should be now updated 🥳

## Developer Checklist

\* Required






Ensure you delete the branch once the Pull Request is merged.
